internal/pkg/agent/cmd/diagnostics.go

```diff
@@ -21,8 +21,8 @@ import (
 func newDiagnosticsCommand(_ []string, streams *cli.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "diagnostics",
-		Short: "Gather diagnostics information from the elastic-agent and write it to a zip archive.",
-		Long:  "Gather diagnostics information from the elastic-agent and write it to a zip archive.",
+		Short: "Gather diagnostics information from the Elastic Agent and write it to a zip archive",
+		Long:  "This command will gather diagnostics information from the Elastic Agent and write it to a zip archive.",
```

Comment:
```suggestion
		Long:  "This command gathers diagnostics information from the Elastic Agent and writes it to a zip archive.",
```
I'd use present tense for consistency.